config.mk

```diff
@@ -32,14 +32,18 @@ CFLAGS_OPT := \
   -flto \
   -fstrict-aliasing
 
+# enable everything and then selectively disable some warnings
 CFLAGS_WARN := \
   -Weverything \
   -Werror \
-  -Wno-unknown-warning-option \
-  -Wno-reserved-id-macro \
+  -pedantic \
+  \
   -Wno-dollar-in-identifier-extension \
   -Wno-gnu-statement-expression \
-  -pedantic
+  -Wno-padded \
+  -Wno-reserved-id-macro \
+  -Wno-unknown-warning-option \
+  -Wno-unused-macros
```

Comment:
Would be nice to eventually have a `cc-option` make macro, but `-Wno-unknown-warning-option` will suffice for now.